LoopKit/GlucoseKit/GlucoseSampleValue.swift

```diff
@@ -10,6 +10,9 @@ public protocol GlucoseSampleValue: GlucoseValue {
     /// Whether the glucose value was provided for visual consistency, rather than an actual, calibrated reading.
     var isDisplayOnly: Bool { get }
 
+    /// Whether the glucose value was entered by the user.
+    var wasUserEntered: Bool { get }
```

Comment:
I really have only one question, and it applies to all of these PRs (so, upon satisfactory answer, they all are fine to me): What is the logic behind adding `wasUserEntered`, and how is it not equal to `!isDisplayOnly`?  The comment on `isDisplayOnly` seems to imply that `!isDisplayOnly` is a calibrated be a (and therefore a user-entered) reading.  There must be a subtlety that I'm not aware of...I may just need to be educated on this.  Thanks!

Reply:
Currently, `isDisplayOnly` really means "used for calibration", which also, at the moment, for the current CGMs is also "user entered". 

However, in order to mitigate the risk of the CGM not reporting data when attempting to bolus, Tidepool Loop will add the ability to allow the user to enter a manual glucose value for the bolus. See https://tidepool.atlassian.net/browse/LOOP-906. (DIY Loop reads glucose values from HealthKit and got around the problem that way, but Tidepool Loop will not read from HealthKit.) 

So, to support this new use case, we have to distinguish a "calibration" glucose value (`isDisplayOnly`) from a "manual" glucose value (`wasUserEntered`), noting that "calibrations" are actually both given the current CGM support.

Does that clear things up?

Reply:
ahhhh, that's really subtle...but it does clear things up, thanks!

Reply:
(In fact, that description was so clear, perhaps it could be added to the comment in this code here? :) )

Reply:
I think `wasUserEntered` is pretty clear (particularly considering it goes along with https://developer.apple.com/documentation/healthkit/hkmetadatakeywasuserentered). I agree that `isDisplayOnly` is confusing and perhaps should have been called `isCalibration` (or similar).